src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -146,6 +146,25 @@ class TransferBalancePage extends React.Component {
      */
     navigateToChooseTransferAccount(filterPaymentMethodType) {
         PaymentMethods.saveWalletTransferMethodType(filterPaymentMethodType);
+
+        // If we only have a single option for the given paymentMethodType do not force the user to make a selection
+        const combinedPaymentMethods = PaymentUtils.formatPaymentMethods(
+            this.props.bankAccountList,
+            this.props.cardList,
+            '',
+            this.props.userWallet,
+        );
+
+        const filteredMethods = _.filter(combinedPaymentMethods, paymentMethod => paymentMethod.accountType === filterPaymentMethodType);
```

Comment:
I forget: underscore.filter will always return an array right? so that the next line `filteredMethods.length` will always exist and not throw an error in the case where maybe nothing gets returned?

Reply:
That's correct! It can handle objects or arrays in the first argument but will always return an array (possibly empty but still will have a length).